models/LEO/README.md

```diff
@@ -0,0 +1,51 @@
+# [LEO](https://arxiv.org/abs/2311.12871) For MMScan Question Answering
+
+
+## Introduction
+
+LEO takes egocentric 2D images, 3D point clouds, and texts as input and
+formulates comprehensive 3D tasks as autoregressive sequence prediction. By instruction-tuning LEO, it extends the capability of LLMs
```

Comment:
Through instruction tuning, LEO extends the capabilities of large language models to unified multi-modal vision-language-action tasks.